.circleci/config.yml

```diff
@@ -157,14 +157,15 @@ jobs:
       - run:
           name: Inspecting with Brakeman
           when: always
-          command: 'bundle exec brakeman -o test-results/brakeman/brakeman.junit -o brakeman.html --no-progress --separate-models'
+          # See https://github.com/presidentbeef/brakeman/issues/1664
+          command: 'bundle exec brakeman -o test-results/brakeman/brakeman.junit -o brakeman.html --no-progress --separate-models --no-exit-on-error' 
```

Comment:
The baked in ruby_parser libs that tokenise ruby for analysis by brakeman aren't currently compatible with ruby 3.1.0 syntax. This has been fixed in a recent ruby_parser commit and I've notified the brakeman team of this. Its just a matter of time until we can remove this and have brakeman being able to parse all of the code with ruby 3.1.0 syntax

Reply:
I say baked in - what this means is that brakeman does not reference the ruby parser as a packaged dependency: it copies the code into its own repository and consumes it as non-vendored libs